detector/severity/pattern_severity.go

```diff
@@ -0,0 +1,46 @@
+package severity
+
+import (
+	"regexp"
+)
+
+type SeverityValue int
+
+const (
+	LowSeverity    SeverityValue = 1
```

Comment:
Can we use iota for this usecase ?

Reply:
I'll make that change 👍

Reply:
changes looks good. Thanks @tt-official  👍